### ISSUE TITLE
Full-app test cannot find npm package

### ISSUE THREAD

**Issue by chrisbolin** [meteor/meteor#7160]
I'm using Meteor 1.3.2.4 (on OS X) and I'd like to add tests to my app. I've installed `practicalmeteor:mocha` [per the guide](http://guide.meteor.com/testing.html) and can successful run and test my app with `meteor test`; however full-app testing fails. To summarize:

**Works** `meteor run`

**Works** `meteor test --driver-package practicalmeteor:mocha`

**Fails** `meteor test --driver-package practicalmeteor:mocha --full-app`

```(STDERR) /Users/chris/.meteor/packages/meteor-tool/.1.3.2_4.16usfmb++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:267
(STDERR)     throw(ex);
(STDERR) Error: Can't find npm module 'react'. Did you forget to call 'Npm.depends' in package.js within the 'modules-runtime' package?```

React is indeed one of my only NPM-style dependencies, but it is in `package.json` (and the app builds and runs fine).

[This question](http://stackoverflow.com/questions/37139035/meteor-test-packages-cant-find-npm-module) describes a similar problem with a _package_ test, not an _app_ test. The solution there is to include the dependency in `package.js`, which I don't have as this is an app not a package.

If this is user error, maybe we should edit the guide? I'm happy to do that.

_(cross-posted here and [SO](http://stackoverflow.com/questions/37601613/meteor-full-app-test-cannot-find-npm-package) in case this is user error)_

**Comment by benjamn** [comment]
This should be fixed in Meteor 1.3.3 (see also https://github.com/meteor/meteor/issues/6827). You can try the beta release of 1.3.3 by running `meteor update --release 1.3.3-beta.2`.

**Comment by chrisbolin** [comment]
sorry, thought that was just package testing
